comment_iowriter.go

```diff
@@ -100,6 +104,8 @@ func (cw *RDJSONLCommentWriter) Post(_ context.Context, c *Comment) error {
 	return nil
 }
 
+func (*RDJSONLCommentWriter) ShouldPrependGitRelDir() bool { return false }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method RDJSONLCommentWriter.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:ChBmZjliMzA0NzFmMGY2YzIxEg9nb2xpbnQtY2lyY2xlY2k= -->